raiden.spec

```diff
@@ -63,7 +63,7 @@ def Entrypoint(dist, group, name, scripts=None, pathex=None, hiddenimports=None,
 # We don't need Tk and friends
 sys.modules['FixTk'] = None
 
-executable_name = 'raiden-{}-{}'.format(
+executable_name = 'raiden-v{}-{}'.format(
```

Comment:
This needs to be updated in the `.travis.yml` file as well, right? 
```yml
file: dist/raiden-${TRAVIS_TAG}-macOS.zip
```

Reply:
No, that's the name of the zip. That always included the `v` prefix. The change above is of the name of the actual executable inside the archive.

Reply:
Ok